base_local_planner/msg/Obstacles.msg

```diff
@@ -0,0 +1,2 @@
+Obstacle nearest
```

Comment:
Are nearest and limiting often different?

Reply:
Nearest is closer to limiting if the object is outside of the half angle